components/runtime/chezscheme/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Richard Lowe
+#
+PREFERRED_BITS=64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		chezscheme
+COMPONENT_VERSION=	9.5.2
+COMPONENT_SUMMARY=	Chez Scheme
+COMPONENT_PROJECT_URL=	https://github.com/cisco/ChezScheme
+COMPONENT_FMRI=		runtime/chezscheme
+COMPONENT_CLASSIFICATION=Development/Other Languages
+COMPONENT_SRC=		ChezScheme-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+  sha256:3a370fdf2ffd67d6a0ccbb993dfab1cbaf4a0a97983c869cfaab40528c33c48b
+COMPONENT_ARCHIVE_URL= \
+  https://github.com/cisco/ChezScheme/archive/v$(COMPONENT_VERSION).tar.gz
+
+# Nanopass
+COMPONENT_NAME_1= nanopass-framework-scheme
+COMPONENT_VERSION_1= 1.9
+COMPONENT_SRC_1= $(COMPONENT_NAME_1)-$(COMPONENT_VERSION_1)
+COMPONENT_ARCHIVE_1= $(COMPONENT_SRC_1).tar.gz
+COMPONENT_ARCHIVE_URL_1= https://github.com/nanopass/$(COMPONENT_NAME_1)/archive/v$(COMPONENT_VERSION_1).tar.gz
+COMPONENT_ARCHIVE_HASH_1=	\
+  sha256:625b239f9030d0b1e86b1fffd8b69f7249a63e8b8ca85195a00cf22889f7fc86
+
+# zlib (configure would get this from github.com/madler/zlib)
+# There seems to be no provision to use a system zlib
+COMPONENT_NAME_2= zlib
+COMPONENT_VERSION_2= 1.2.11
+COMPONENT_SRC_2= $(COMPONENT_NAME_2)-$(COMPONENT_VERSION_2)
+COMPONENT_ARCHIVE_2= $(COMPONENT_SRC_2).tar.gz
+COMPONENT_ARCHIVE_URL_2= https://zlib.net/$(COMPONENT_ARCHIVE_2)
+COMPONENT_ARCHIVE_HASH_2=	\
+  sha256:c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
+
+# stex
+COMPONENT_NAME_3= stex
+COMPONENT_VERSION_3= 1.2.1
+COMPONENT_SRC_3= $(COMPONENT_NAME_3)-$(COMPONENT_VERSION_3)
+COMPONENT_ARCHIVE_3= $(COMPONENT_SRC_3).tar.gz
+COMPONENT_ARCHIVE_URL_3= https://github.com/dybvig/stex/archive/v$(COMPONENT_VERSION_3).tar.gz
+COMPONENT_ARCHIVE_HASH_3=	\
+  sha256:bf784ca46aaca9b665b7eb0c39f04f6a695aa40e99b11d8a6d4440648c1bf40e
+
+COMPONENT_LICENSE=	Apache
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+PATH=$(PATH.gnu)
```

Comment:
Put it after `COMPONENT_ARCHIVE_URL` so it's clear it belongs to the main component not to bundles.